stage0/gc.go

```diff
@@ -125,9 +164,20 @@ func ForceMountGC(path, uuid string) error {
 		return fmt.Errorf("error getting mounts for pod %s from mountinfo: %v", uuid, err)
 	}
 
+	for i := len(mnts) - 1; i >= 0; i -= 1 {
+		mnt := mnts[i]
+		if strings.Contains(mnt.mode, "shared") || strings.Contains(mnt.mode, "master") {
```

Comment:
This is to say - I would probably use either `[]string` or `map[string]struct{}` for mode. Or, if you really want it to be a string then it should be in format like `,shared,master,blahblah,some-other-mode,`, so you can call `strings.Contains(mnt.mode, ",shared,")` instead.